app/lib/katello/validators/gpg_key_content_type_validator.rb

```diff
@@ -5,7 +5,7 @@ def validate(record)
         # need to verify, that gpg_key is of GpgKey.content type "gpg_key" and
         # ssl_ca_cert, ssl_client_cert, ssl_client_key of GpgKey.content type "cert"
 
-        if !record.gpg_key.blank? && record.gpg_key.content_type != "gpg_key"
+        if !record.gpg_key_id.blank? && record.gpg_key.content_type != "gpg_key"
```

Comment:
Why did this have to change? /me probably missing something obvious :)

Reply:
Something else that needed to be changed that errored out after I changed the table name with the migration. @jlsherrill do you know why?

Reply:
Does the unit test you wrote fail without it?

Reply:
My unit test passed with changing it back to `gpg_key` so maybe I misread something. I am letting the rest of the test suite run, if everything passes, I will push up the updated fix.

Reply:
I think this should work either way, you might have just changed it when you were debugging the table name issues +1 to changing it back

Reply:
Updated